Gemfile.lock

```diff
@@ -113,4 +113,4 @@ DEPENDENCIES
   wdm (~> 0.1.0)
 
 BUNDLED WITH
-   2.1.2
+   1.17.3
```

Comment:
Please do not include changes unless there's an intention to do so.

Reply:
ok

Reply:
In case I wasn't clear, please remove this.

Reply:
Removed!